src/prefect/cli/templates/kubernetes-orion.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: prefect-orion
```

Comment:
What do you think about just calling the Deployment and Service `orion`?  I think a common use case will be:

* Run the Orion server in a `prefect` namespace in your cluster
* Run multiple agents in different namespaces

So if the name of the service is `prefect-orion`, then the API hostname might be `prefect-orion.prefect.svc.cluster.local`. I think it might be cleaner if it were `orion.prefect.svc.cluster.local` (but this is just one opinion).

Reply:
I agree. I just kept the default of Prefect 1.0.

Reply:
Let's consider this in a follow-up. Can you create an issue Chris?

Reply:
https://github.com/PrefectHQ/prefect/issues/6810 Again, this is a weakly-held preference ;)

Reply:
@hallenmaia No need to hold up this PR over that, we have the future issue to consider.